service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseValidatorService.java

```diff
@@ -47,7 +48,7 @@ public CaseValidatorService(Validator validator) {
         List<String> errorList;
 
         errorList = caseData.stream()
```

Comment:
Would rename `caseData` to `constraintViolations` or something like that, `caseData` isn't accurate in describing what that variable contains.